renovate-presets/gradle.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Default preset for use with Jellyfin's JVM repos",
+  "extends": ["github>jellyfin/.github//renovate-presets"],
+  "packageRules": [
+    {
+      "description": "Add the java GitHub label to Gradle dependency bump PRs",
+      "matchManagers": ["gradle"],
+      "labels": ["java"]
```

Comment:
java? how dare you 👀

Reply:
you want me to label them JVM, Kotlin, Gradle? I am all :ear: (I will accept everything BUT GROOVY)

Reply:
We don't do groovy either :p

Since Gradle is both the build system & dependency manager I think it makes sense to call it gradle I guess.

Reply:
actually you are right, I add the NuGet label to NuGet Packages, so I could add Gradle as the label instead, even if it is not really accurate :thinking: 
Aren't Gradle deps just Maven packages? (jk, I would never label Gradle dep PRs with Maven, only monsters would do such heresy)

Reply:
Gradle supports multiple repositories (unlike npm 🤷) and we use both the Maven Central and Google repositories.

Reply:
I am aware of that, as far as I know Gradle could even support C++ (not sure how they handle dependencies there though).
The `gradle` label works fine IMHO and is reasonable.
Additionally, you can always add more labels in the corresponding repository configs of Renovate :slightly_smiling_face: